lib/processors/bundlers/moduleBundler.js

```diff
@@ -37,6 +37,55 @@ class LocatorResourcePool extends ResourcePool {
 	}
 }
 
+/**
+ * A ModuleBundleDefinitionSection specifies the embedding technology ('provided', 'raw', 'preload' or 'require')
+ * and lists the resources that should be in- or excluded from the section.
+ * <p>
+ * <b>Module bundle section modes</b><br>
+ * <ul>
+ * 	<li>
+ * 		<code>provided</code>: A section of type 'provided' defines a set of modules that should not be included in the bundle file itself, but
+			which should be assumed to be already loaded (or 'provided') by the environment into which bundle module is loaded.
+ * 	</li>
+ *	<li>
+		 <code>raw</code>: A raw section determines the set of modules that should be embedded,
+			sorts them according to their dependencies and writes them out 1:1 without any transformation or wrapping (raw). Only JavaScript sources
+				can be embedded in a raw section.
+	</li>
+ *	<li>
+		 <code>preload</code>: A preload section packages resources that should be stored in the preload cache in the client.
+			They can embed any textual resource type (JavaScript, CSS and Theme-Parameter files) that the bundling supports. The modules are wrapped into a call to jQuery.sap.registerPreload()
```

Comment:
This depends on the bundle format. In EVO-format. sap.ui.predefine us used for pure AMD modules, sap.ui.require.preload for other modules. In legacy  UI5 format, sap.ui.predefine is also used for AMD modules, but jQuery.sap.registerPreload is used for other resources.

Reply:
Done